pricing-page-exercise/mesut/style.css

```diff
@@ -0,0 +1,342 @@
+* {
+    box-sizing: border-box;
+    font-family: "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
+}
+
+body {
+    margin: 0;
+    
+}
+
+
+.navbar {
+    height: 70px;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    border-bottom: solid 1px #dfdfdf;
+    padding: 16px 24px;
+    margin-bottom: 16px;
+    box-shadow: 0px 2px 10px #a2a0a04c;
+}
+
+.top-right-links {
+    display: flex;
+    justify-content: end;
+    align-items: center;
+
+}
+
+.top-right-buttons {
+    display: flex;
+    align-items: center;
+    list-style-type: none;
+    color: #1D2124;
+    font-size: 16px;
+}
+
+.top-right-list {
+    padding: 12px;
+    color: #1d2124;
+
+}
+
+.company-link {
+    padding: 10px;
+    font-size: 20px;
+    color: #212529;
+    white-space: nowrap;
+    justify-content: start;
+}
+
+
+.link {
+    text-decoration: none;
+}
+
+
+.top-right-buttons:visited {
+    color: #1D2124;
+}
+
+.features-button:hover {
+    text-decoration: underline;
+}
+
+.enterprise-button:hover {
+    text-decoration: underline;
+}
+
+.support-button:hover {
+    text-decoration: underline;
+}
+
+.pricing-button:hover {
+    text-decoration: underline;
+}
+
+
+.signup-button {
+    color: #007BFF !important;
```

Comment:
Avoid using `!important` as much as possible. In this case, you can use to classes to make a more specific selection `.top-right-buttons .signup-button`.